style.css

```diff
@@ -810,14 +865,147 @@ nav ul.header-links li.header-link:hover {
     resize: both;
     overflow: auto;
     padding: 25px;
+    line-height: 25px;
   }
 
   .submit-button {
-    margin: 35px;
+    margin: 35px 175px 35px 0;
     border: none;
   }
 
   .email-id {
     display: none;
   }
+
+  .contact-form {
+    display: flex;
+    flex-direction: row;
+    padding: 60px;
+    padding-bottom: 39px;
+    margin-bottom: -390px;
+  }
```

Comment:
- [ ] I noticed that the suggestion from the previous reviewer **hasn't been implemented yet.** Could you please add the background image correctly to your contact section? Currently, the section appears a bit small and there are two empty spaces on the left and right. 
To help you understand better, I've created a video demonstrating how to solve a similar problem with your main section. You can apply the same approach to the contact section and align it more closely with the provided [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468). 😉

   **_Here's the video for reference._**
   
   [17_2023_2-05-53 PM.webm](https://github.com/DevKaranJ/MyPortfolio/assets/107952724/ed05d13c-cf23-4db9-aea7-892fb38f76c6)

   ### Your design:
   ![image](https://github.com/DevKaranJ/MyPortfolio/assets/107952724/c037a63a-a515-484f-9284-0da7ab5e3675)

   ### Figma design:
   ![image](https://github.com/DevKaranJ/MyPortfolio/assets/107952724/ae163e8f-33c3-45b7-aac9-17aadc88a68d)